### PR TITLE
Remove self

### PR DESCRIPTION
**Pull request by marijnschilling**
And it's gone💢 

Just added some question when I do not understand when the compiler needs `self` after all

### CODE REVIEW

File: Sources/CountLabel.swift [hunk 1]
```diff
@@ -48,15 +48,15 @@ open class CountLabel: UILabel {
...
-        self.progress = 0
-        self.totalTime = duration
-        self.lastUpdate = Date.timeIntervalSinceReferenceDate
+        easingRate = Double(abs(endValue - startValue)).map(0 ... 9999, 1.1 ... 3.0)
+        span = abs(endValue - startValue)
+        progress = 0
+        totalTime = duration
+        lastUpdate = Date.timeIntervalSinceReferenceDate
 
         let timer = CADisplayLink(target: self, selector: #selector(updateValue))
```
Comment (marijnschilling) [inline Sources/CountLabel.swift]: So I guess I need to use self here cause of this local property, but what is the best practise? To give the local property another name, or to use self for the stored property?

File: Demo/ViewController.swift [hunk 2]
```diff
@@ -4,7 +4,7 @@ import CountLabel
 class ViewController: UIViewController {
 
     lazy var countLabel: CountLabel = {
-        let countLabel = CountLabel(frame: CGRect(x: 22 , y: 50, width: self.view.bounds.width - 44, height: 200))
+        let countLabel = CountLabel(frame: CGRect(x: 22 , y: 50, width: self.view.bounds.width - 44, height: 200))  //Why do i need self here?
```
Comment (marijnschilling) [inline Demo/ViewController.swift:7]: Why do I need self here to access the view? 🤔

File: Sources/CountLabel.swift (hunk 1 above)
Comment (yuliaveres) [reply Sources/CountLabel.swift]: i would use another name in general;

and why u need this local variable ? can't work with global timer ?

Comment (roberthein) [reply Sources/CountLabel.swift]: You can also assign to `timer` directly, without storing it in a new local variable first.

File: Demo/ViewController.swift (hunk 2 above)
Comment (marijnschilling) [reply Demo/ViewController.swift:7]: Aha because lazy var is a closure ofc ✌️

Comment (elland) [reply Demo/ViewController.swift:7]: Good question, no idea. What does the compiler say?

File: Sources/CountLabel.swift [hunk 3]
```diff
@@ -44,56 +44,55 @@ open class CountLabel: UILabel {
         self.startValue = startValue
         self.endValue = endValue
 
-        self.timer?.invalidate()
-        self.timer = nil
-
+        timer?.invalidate()
+        timer = nil
+
```
Comment (elland) [inline Sources/CountLabel.swift:49]: Did you remove self manually instead of with `swiftformat`?

File: Demo/ViewController.swift (hunk 2 above)
Comment (elland) [reply Demo/ViewController.swift:7]: Right!

File: Sources/CountLabel.swift (hunk 3 above)
Comment (marijnschilling) [reply Sources/CountLabel.swift:49]: yeah manually

Comment (marijnschilling) [reply Sources/CountLabel.swift:49]: Does swift format remove it? Or can i change the settings for it?

Comment (elland) [reply Sources/CountLabel.swift:49]: It removes by default even. 👍

**Pull request merged**